src/content/concepts/entry-points.md

```diff
@@ -68,6 +68,41 @@ Below is a list of entry configurations and their real-world use cases:
 
 ### Separate App and Vendor Entries
 
+__webpack.config.js__
+
+```javascript
+module.exports = {
+  entry: {
+    main: './src/app.js',
+    vendor: './src/vendor.js'
+  }
+};
+```
+
+__webpack.prod.js__
+
+```javascript
+module.exports = {
+  output: {
+    filename: [name].contentHash.bundle.js
```

Comment:
Please use valid JavaScript.

Reply:
filename: [name].[contentHash].bundle.js
This fails travis CI test. @montogeek